python/ray/tune/suggest/ax.py

```diff
@@ -230,8 +230,11 @@ def _setup_experiment(self):
             logger.warning("Detected sequential enforcement. Be sure to use "
                            "a ConcurrencyLimiter.")
 
-    def set_search_properties(self, metric: Optional[str], mode: Optional[str],
-                              config: Dict):
+    def set_search_properties(self,
+                              metric: Optional[str],
+                              mode: Optional[str],
+                              config: Dict,
+                              experiment: Optional[Dict] = None):
```

Comment:
Maybe we can do
```suggestion
                              **spec):
```
instead